### PR TITLE
FE good stuff

### PR DESCRIPTION
**Pull request by hlyford11**
- flashy dealie
- ‘also might like’ section when quiz ends (awaiting logic for the list)
- player card style improvements

### CODE REVIEW

File: public/javascripts/quiz.js [hunk 1]
```diff
@@ -156,7 +156,10 @@ var addToCorrectList = function (count, player, index){
   if (player.guessed == true) return;
   var html = '<div class="outer-guess"><div class="correct-guess" data-index="'+index+'">'+count +'.' + '<img class="circle-pic" src='+player.avatar_url+'>' + player.full_name + '</div></div>';
   answer_container.append(html);
-}
+  var current = answer_container.find('.correct-guess[data-index="' + index + '" ]').parent();
```
Comment (blakeparkinson) [inline public/javascripts/quiz.js]: noticed how I changed the selector here to ```answer_container```. This allows for less DOM scanning

File: public/stylesheets/style.css [hunk 2]
```diff
@@ -376,7 +398,40 @@ padding-left: 3%;
...
+    animation-delay: 1s;
+    -moz-animation: fadeout 2s forwards;
+    -moz-animation-delay: 1s;
+    -webkit-animation: fadeout 2s forwards;
+    -webkit-animation-delay: 1s;
+    -o-animation: fadeout 2s forwards;
+    -o-animation-delay: 1s;
+}
+
+@keyframes fadeout {
```
Comment (blakeparkinson) [inline public/stylesheets/style.css]: True GOATS (me) use badass css animations instead of crappy js hax

**Comment by blakeparkinson** [comment]
looks good with @blakeparkinson 's updates. Hes so dreamy

**Comment by hlyford11** [comment]
stack overflow liked how i did it :-/

On Sun, Apr 19, 2015 at 10:54 PM, Blake Parkinson <notifications@github.com>
wrote:

> looks good with @blakeparkinson <https://github.com/blakeparkinson> 's
> updates. Hes so dreamy
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blakeparkinson/sports2/pull/284#issuecomment-94364294>
> .
>



-- 
*Henry Lyford*
Manager, Partnerships & Business Development
Edmodo.com/henry

**Pull request merged**